index.js

```diff
@@ -0,0 +1,49 @@
+function initMenu() {
+  // get the hamburger element
+  const hamburger = document.querySelector('#hamburger');
+  // get the close-btn element
+  const closeBtn = document.querySelector('#close-hamburger');
+  // get the menu-item elements
+  const links = document.querySelectorAll('.menu-item a');
+  // get the mobile menu container element
+  const mobileMenu = document.querySelector('.mobile-menu');
+  // Kill the nav menu
+  function destroyNavMenu() {
+    mobileMenu.style.display = 'none';
+  }
+  // toggle visibility function for nav menu
+  function toggleNavMenu() {
+    mobileMenu.classList.toggle('show-menu');
+  }
+  // open mobile nav container
+  if (hamburger) {
+    hamburger.addEventListener('click', toggleNavMenu);
+  }
+  // close mobile nav container
+  if (closeBtn) {
+    closeBtn.addEventListener('click', toggleNavMenu);
+  }
+  // navigate to the actual links on the menu item(s) - li a
+  if (links) {
+    links.forEach((link) => {
+      // add event listener to every link  - li a
```

Comment:
- [ ] Great job fixing the issue mentioned by the previous reviewer. but you need to fix the below issue.
  - After opening the mobile menu and clicking on any menu option it closes the menu but after that, I'm not able to open the mobile menu again. Please make sure to fix this issue.

Reply:
@thecodechaser thank you for your observation. I have fixed the issue.